source/chapters/configuration.rst

```diff
@@ -160,31 +160,36 @@ computer. **Therefore it is important to take care to choose the best Audio API
 available to you.** Refer to the following table of Audio APIs to see what the
 best choice is for your operating system.
 
-+----------------------------------------+--------------+
-| OS / Audio API                         | Quality      |
-+========================================+==============+
-| Windows / WMME                         | Poor         |
-+----------------------------------------+--------------+
-| Windows / DirectSound                  | Poor         |
-+----------------------------------------+--------------+
-| Windows / WASAPI                       | Good         |
-+----------------------------------------+--------------+
-| Windows / ASIO                         | Good         |
-+----------------------------------------+--------------+
-| Windows / WDDKMS                       | Good         |
-+----------------------------------------+--------------+
-| Mac OS X / CoreAudio                   | Good         |
-+----------------------------------------+--------------+
-| GNU Linux / OSS                        | OK           |
-+----------------------------------------+--------------+
-| GNU Linux / ALSA                       | Good         |
-+----------------------------------------+--------------+
-| GNU Linux / JACK (Advanced)            | Good         |
-+----------------------------------------+--------------+
-
-On Windows, if an ASIO driver is not available for your operating system, you
-can try installing `ASIO4ALL <http://asio4all.com>`_, a low-latency audio driver
-for WDM audio devices.
++-----------------------------+---------+
+| OS / Audio API              | Quality |
++=============================+=========+
+| Windows / ASIO              | Good    |
++-----------------------------+---------+
+| Windows / WDM-KS            | Good    |
++-----------------------------+---------+
+| Windows / WASAPI            | OK      |
++-----------------------------+---------+
+| Windows / DirectSound       | Poor    |
++-----------------------------+---------+
+| Windows / MME               | Poor    |
++-----------------------------+---------+
+| Mac OS X / CoreAudio        | Good    |
++-----------------------------+---------+
+| GNU Linux / ALSA            | Good    |
++-----------------------------+---------+
+| GNU Linux / JACK (Advanced) | Good    |
++-----------------------------+---------+
+| GNU Linux / OSS             | OK      |
++-----------------------------+---------+
+
+On Windows, if your soundcard comes with a an ASIO driver bypassing the 
+kernel, it is the best choice to achieve a low latency. 
+
+For an ASIO Setup, every soundcard need to provide its own ASIO driver.
```

Comment:
I don't think this sentence is necessary. I think this paragraph can be combined with the one above.